scripts/post-team-sync.py

```diff
@@ -17,71 +17,65 @@
 # %%
 from ghapi.all import GhApi
 from IPython.display import Markdown
-from textwrap import dedent
 from datetime import date
 from ghapi.actions import github_token
 import os
+from pathlib import Path
 
+# %% [markdown]
+# # Priority issues
+
+# %%
 # On GitHub Actions "ACCESS_TOKEN" should be a personal access token with r/w permissions to *other* repos
 token = github_token() if "ACCESS_TOKEN" not in os.environ else os.environ["ACCESS_TOKEN"]
 
-# %%
 # Initialize the GH API and our markdown
 api = GhApi(token=token)
 
-# %% [markdown]
-# # Base template
-# The base template is defined here: https://github.com/2i2c-org/team-compass/blob/main/.github/ISSUE_TEMPLATE/team-update.md
-# It has placeholders for lists of issues, and these will be automatically filled in below, then a new issue will be created.
-
 # %%
-# Grab our issue template
-from base64 import b64decode
-template = api.repos.get_content("2i2c-org", "team-compass", ".github/ISSUE_TEMPLATE/team-update.md")
-template = b64decode(template.content).decode("utf-8")
-
-# This removes the header bracketed by ---
-template = "---".join(template.split("---")[2:]).strip()
+# Grab Ready to Work cards that point to issues of high priority
+boards = api.projects.list_for_org("2i2c-org")
+priority_labels = ['prio: high', 'priority']
+
+priority_issues = []
+for board in boards:
+    if "Activity" in board.name:
```

Comment:
So, I guess you are thinking here the Activity board content is too much fine-grained to be on this update, right?

Reply:
yeah I feel like this issue is more like "the kinds of things I plan to focus on" rather than "the specific thing I'm doing tomorrow". That make sense?

Reply:
Total sense to me.